shell/browser/renderer_host/electron_render_message_filter.cc

```diff
@@ -40,13 +44,22 @@ void EmitPreconnect(content::BrowserContext* browser_context,
   }
 }
 
+void ForwardCursorChange(mate::Handle<WebContents> web_contents,
+                         const content::WebCursor& cursor) {
+  if (!web_contents.IsEmpty()) {
+    web_contents->OnCursorChange(cursor);
+  }
+}
+
 }  // namespace
 
 ElectronRenderMessageFilter::ElectronRenderMessageFilter(
-    content::BrowserContext* browser_context)
+    content::BrowserContext* browser_context,
+    mate::Handle<WebContents> web_contents)
```

Comment:
`mate::Handle` doesn't seem like the right type to take here. I think we should use `WeakPtr<WebContents>` instead. (It might be that just `WebContents*` is fine... but i'm not totally sure that the WebContents is guaranteed to outlive the RenderMessageFilter.)